terraform/azure/inventory.tmpl

```diff
@@ -5,15 +5,15 @@ all:
 %{ for index, value in hana-pip ~}
         ${hana-name[index]}:
           ansible_host: ${value}
-          ansible_python_interpreter: /usr/bin/python3
+          %{ if hana-major-version == "12" }ansible_python_interpreter: /usr/bin/python2.7%{ else }ansible_python_interpreter: /usr/bin/python3 %{ endif }
 %{ endfor ~}
  %{ if iscsi-enabled }
     iscsi:
       hosts:
  %{ for index, value in iscsi-pip ~}
         ${iscsi-name[index]}:
           ansible_host: ${value}
-          ansible_python_interpreter: /usr/bin/python3
+          %{ if iscsi-major-version == "12" }ansible_python_interpreter: /usr/bin/python2.7%{ else }ansible_python_interpreter: /usr/bin/python3%{ endif }
 %{ endfor ~}
 %{ endif }
-  hosts: null
\ No newline at end of file
+  hosts: null
```

Comment:
ansible_python_path is now based on the os_image used or by a specific override in tfvars.  SLES 12 will use python2.7 and 15 will use python3

Reply:
Quite like, needs to be ported to other Cloud providers templates too, right?

Reply:
Yes. This is only for Azure at the moment